browser-test/src/support/ti_dashboard.ts

```diff
@@ -33,19 +33,20 @@ export class TIDashboard {
   }
 
   async expectDashboardContainClient(client: ClientInformation) {
-    const rowText = await this.page
-      .locator(
-        `.cf-admin-question-table-row:has-text("${client.emailAddress}")`,
-      )
-      .innerText()
-    expect(rowText).toContain(this.convertToMMDDYYYY(client.dobDate))
+    const row = this.page.locator(
+      `.cf-admin-question-table-row:has-text("${client.emailAddress}")`,
+    )
+    const rowText = await row.innerText()
     expect(rowText).toContain(client.firstName)
     expect(rowText).toContain(client.lastName)
+    // date of birth rendered as <input> rather than plain text.
+    expect(await row.locator('input[name="dob"]').inputValue()).toEqual(
+      client.dobDate,
+    )
   }
 
   async expectDashboardNotContainClient(client: ClientInformation) {
     const tableInnerText = await this.page.innerText('table')
-    expect(tableInnerText).not.toContain(this.convertToMMDDYYYY(client.dobDate))
```

Comment:
Can this check not be maintained?

Reply:
Not easily. We need to check that table has no <input> fields with value equal to the date of birth. It gets very specific and britle. Checking that table doesn't contain email and name should be sufficient to ensure that client doesn't is not rendered.